plumpy/process_states.py

```diff
@@ -342,14 +344,17 @@ class Excepted(State):
     TRACEBACK = 'traceback'
 
     def __init__(
-        self, process: 'Process', exception: Optional[BaseException], trace_back: Optional[TracebackType] = None
+        self,
+        process: 'Process',  # pylint: disable=unused-argument
+        exception: Optional[BaseException],
+        trace_back: Optional[TracebackType] = None
     ):
         """
         :param process: The associated process
         :param exception: The exception instance
         :param trace_back: An optional exception traceback
         """
-        super().__init__(process)
+        super().__init__(None)  # terminal state does not require process ref
```

Comment:
Could you just elaborate on this comment a litle to say 
'Terminal states don't get a backreference to the state machine so that it's easier for python to clear the memory'
or something to that effect just so that it's clear:
1) Why this was done, and,
2) So it's clear under which circumstances it's `None` (which you've already stated actually)